just.py

```diff
@@ -125,14 +156,33 @@ def executeTasks(cmd_args, tasks):
 
     qsub_id = None
     prologue_body = []
-    for task_id in sorted(tasks.keys()):
 
-        task_name = tasks[task_id]['task_name']
-        task_body = tasks[task_id]['task_body']
-        if task_id == 0:
-            prologue_body = task_body
+    if 0 in tasks:
+        prologue_body = tasks[0]['task_body']
 
-        elif start <= task_id <= stop:
+    queue = dict()
+    for child_id in task_parents:
+        if start <= child_id <= stop:
+            parents = task_parents[child_id]
+            range_parents = []
+            for parent in parents:
+                if start <= parent <= stop:
+                    range_parents.append(parent) # Make sure only range is viewed ... could fail in unexpected ways
+            queue[child_id] = range_parents
```

Comment:
could write: `queue[child_id] = [p for p in task_parents[child_id] if start <= p <= stop]`